GOVERNANCE.md

```diff
@@ -9,6 +9,16 @@ It extends [Technical Charter](https://stackstorm.com/wp/wp-content/uploads/2020
 The Technical Steering Committee is a group of Maintainers (Committers) who have earned the ability to modify (“commit”) sourcecode, documentation or other artifacts.
 This group is responsible for all technical oversight of StackStorm as Open Source Project.
 
+### TSC Meetings
+Because of the different schedules and time zones, TSC is keen to prioritize asynchronous collaboration and work.
+However, every `1st Tuesday` of the month `09:30 AM US Pacific`, [@StackStorm/tsc](OWNERS.md) members are gathering for `1 hour` to discuss the project problems, roadmap, priorities to help the StackStorm as a project evolve and succeed.
```

Comment:
Let's change this to say that we have a TSC meeting every month. Please see https://github.com/StackStorm/community/issues/33 for the schedule.

Reply:
Thanks, updated in https://github.com/StackStorm/st2/pull/5711/commits/534e604a0375411c262a8e0393d49eaa25751a53